e2e/cypress/integration/interactive_dialog/full_dialog_spec.js

```diff
@@ -104,6 +105,16 @@ describe('ID15888 Interactive Dialog', () => {
 
                     // * Verify that the default value is the first element of the list
                     cy.wrap($elForm).find('input').first().should('have.value', 'engineering').and('have.attr', 'checked');
+                } else if (element.name === 'boolean_input') {
+                    cy.wrap($elForm).find('.checkbox').should('be.visible').within(() => {
+                        const checkedState = element.default === 'True' ? 'be.checked' : 'be.not.checked';
```

Comment:
nit: The [default](https://github.com/mattermost/mattermost-webapp/pull/3924/files#diff-7c6068d798da938f85263ae27622e1d6R164) is `True` and so it should always be `be.checked`.  This should be fine, however, my worry is in case a bug is introduced in server (unintentionally flip the logic), this will not be able to detect it.

Reply:
Changed to just check for the checked state.